constants/index.ts

```diff
@@ -0,0 +1,27 @@
+// File for all project constant
+
+export const PROFILE_PATH = '/profile'
+export const CURRICULUM_PATH = '/curriculum'
+export const REVIEW_PATH = '/review'
+export const REPO_PATH = 'https://github.com/garageScript/c0d3-app'
+export const ADMIN_PATH = '/admin'
+export const DOCS_PATH = '/docs'
+export const DISCORD_PATH = 'https://discord.gg/c0d3'
+export const LOGIN_PATH = '/login'
+export const SIGNUP_PATH = '/signup'
+
+// URL Constants
+export const DEPLOYMENT_URL = process.env.VERCEL_URL
+  ? `https://${process.env.VERCEL_URL}`
+  : `http://localhost:${process.env.PORT ?? 3000}`
```

Comment:
Is there a need to explicitly check for null or undefined using `??` operator? Seems like it may be unneeded additional complexity. If the OR operator `||` can be used here instead it would make things simpler

Reply:
Yep, you are right `||` would be better in this case, I have updated it.